logger/log.go

```diff
@@ -67,15 +68,26 @@ func NewFileLogger(filename string, time, debug, trace, pid bool) *Logger {
 	}
 
 	l := &Logger{
-		logger: log.New(f, pre, flags),
-		debug:  debug,
-		trace:  trace,
+		logger:  log.New(f, pre, flags),
+		debug:   debug,
+		trace:   trace,
+		logFile: f,
 	}
 
 	setPlainLabelFormats(l)
 	return l
 }
 
+// Close cleans up any resources with the server's logger implementation.
+func (l *Logger) Close() {
```

Comment:
Might be better if it returns an error here so that it also implements `io.Closer`: https://golang.org/pkg/io/#Closer